dataland-frontend/tests/e2e/fixtures/lksg/LksgPreparedFixtures.ts

```diff
@@ -66,13 +81,186 @@ function manipulateFixtureForOneLksgDataSetWithProductionSites(input: FixtureDat
   return input;
 }
 
+function manipulateFixtureForVat20231(input: FixtureData<LksgData>): FixtureData<LksgData> {
+  input.companyInformation.companyName = "vat-2023-1";
+  input.t.social!.general!.vatIdentificationNumber = "2023-1";
+  input.t.social!.general!.dataDate = "2023-04-18";
+  input.reportingPeriod = "2023";
+  return input;
+}
+
+function manipulateFixtureForVat20232(input: FixtureData<LksgData>): FixtureData<LksgData> {
+  input.companyInformation.companyName = "vat-2023-2";
+  input.t.social!.general!.vatIdentificationNumber = "2023-2";
+  input.t.social!.general!.dataDate = "2023-06-22";
+  input.reportingPeriod = "2023";
+  return input;
+}
+
+function manipulateFixtureForVat2022(input: FixtureData<LksgData>): FixtureData<LksgData> {
+  input.companyInformation.companyName = "vat-2022";
+  input.t.social!.general!.vatIdentificationNumber = "2022";
+  input.t.social!.general!.dataDate = "2022-07-30";
+  input.reportingPeriod = "2022";
+  return input;
+}
+
 /**
- * Sets the company name in the fixture data to a specific string
+ * Generates a new LKSG fixture without any undefined values
  *
  * @param input Fixture data to be manipulated
  * @returns the manipulated fixture data
  */
-function manipulateFixtureForTwoDifferentDataSetTypes(input: FixtureData<LksgData>): FixtureData<LksgData> {
-  input.companyInformation.companyName = "two-different-data-set-types";
+function manipulateFixtureToContainEveryField(input: FixtureData<LksgData>): FixtureData<LksgData> {
```

Comment:
Technically this does not need any input. It could simply return the constructed model instead of overwritting every field.

Reply:
This method is used in a loop, which loops through all "manipulateFixtureFor..." methods. 
All other "manipulateFixtureFor..." methods accept an input, therefore this one here is forced to require an input, otherwise it cannot be used in that loop.
It's not very beautifuly.  Therefore if you are not happy with it, we can fix it in the version history ticket here.

Reply:
It needs the input to derive the company information from it.